clouddrift/binning.py

```diff
@@ -33,12 +329,21 @@ def binned_statistics(
         - None: defaults to 10 bins per dimension.
     bins_range : list of tuples, optional
         Outer bin limits for each dimension.
+    statistics : str or list of str, Callable[[np.ndarray], float] or list[Callable[[np.ndarray], float]]
+        Statistics to compute for each bin. It can be:
+        - a string, supported values: 'count', 'sum', 'mean', 'median', 'std', 'min', 'max', (default: "count"),
+        - a custom function as a callable that take a 1D array of values and return a single value.
```

Comment:
a custom function as a callable for univariate statistics that take a 1D array of values and return a single value.